apps/storybook/src/ComboBox.stories.tsx

```diff
@@ -575,3 +576,36 @@ Virtualized.args = {
 Virtualized.argTypes = {
   value: { control: { disable: true } },
 };
+
```

Comment:
After deselecting all items, I get this error:
![image](https://user-images.githubusercontent.com/36186912/191184166-dc3c5c2c-765e-43ac-9fbf-9cc305cfbdab.png)

Reply:
I think I fixed that. I was returning undefined instead of empty array.